backend/src/main/java/ca/bc/gov/hlth/mohums/exceptions/ErrorResponse.java

```diff
@@ -0,0 +1,51 @@
+package ca.bc.gov.hlth.mohums.exceptions;
+
+import java.time.LocalDateTime;
+
+public class ErrorResponse {
```

Comment:
My idea here was to build a base for exception handling - if we were to add more endpoints in the future, that do not serve only as Keycloak proxy. `@ControllerAdvice` is a global entry point for exception handling, so my thought was that it would contain all the exception handling methods for Controllers. That's why the generic class names. I should probably rename `handleValidationExceptions` as this method applies to validating BulkRemovalRequests specifically. 

This might have been me over-engineering things again, though. I could just move the `handleValidationExceptions` to `UsersController` and get rid of the global exception handling class.

Please let me know if I should:
1. Simplify this and move this `handleValidationExceptions` to `UsersController`  and rename the method.
2. Leave the class annotated with `@ControllerAdvice` and rename method inside of it. In that case I would like to leave the generic `ErrorResponse` and `ControllerExceptionHandler` names and rename the method inside of it.